lib/matplotlib/backends/backend_qt5.py

```diff
@@ -268,12 +268,9 @@ def _update_figure_dpi(self):
     @property
     def _dpi_ratio(self):
         # Not available on Qt4 or some older Qt5.
-        try:
-            ratio = self.devicePixelRatio()
-            if ratio == 0:
-                return 1
-            else:
-                return ratio
+            try:
```

Comment:
Indentation error

Reply:
I'm sorry, I should have double checked that...